### PR TITLE
Update all.json

### PR DESCRIPTION
**Pull request by frankywild**
|Site    |Urlscan|
|-------|-------|
|app-polkastarter.live|	https://urlscan.io/result/3c9398a3-6f73-4253-8c61-099dd6f2b673/|
|itheum.biz|	https://urlscan.io/result/e7cb06b1-93c3-4f94-9b55-53fe44592bdf/|
|poikastarter.dev|	https://urlscan.io/result/51353fda-b97c-4b7c-82d3-a9d7d5cf03a2/|
|polcastarter.com|	https://urlscan.io/result/5f9fbdc4-591c-4b20-80ed-668643017560/|
|polkaprojects-interface.online|	https://urlscan.io/result/8e9537c9-df00-4bf7-b14f-82e97844113e/|
|polkaslarterlercom.info|	https://urlscan.io/result/fb5eedf1-2e45-487d-b922-e582be0b68d4/|
|polkaslarterlercom.live|	https://urlscan.io/result/a25c9647-88e8-4a70-b5fa-74c5c9901d99/|
|polkastartar.live|	https://urlscan.io/result/051c3931-36ab-46ca-b0a2-bcf308e355de/|
|polkastartel.com|	https://urlscan.io/result/95a96152-8b0d-4f66-b9f6-baed529e76bb/|
|polkastarter-projects.com|	https://urlscan.io/result/78c2138d-2e8a-4e13-8395-e90df91726f6/|
|polkastarterweb.org|	https://urlscan.io/result/1dab0dca-e91f-4bd6-953e-ebbd8dc0636c/|
|rampad.org|	https://urlscan.io/result/8f5afbdc-a6f4-4541-9866-87252d41c508/dom/|

### CODE REVIEW

File: all.json [hunk 1]
```diff
@@ -9,6 +9,19 @@
 		"12345678987654321.link",
 		"16-b.it",
 		"162.222.226.37",
+		"v2.polkastarter.com",
```
Comment (jacogr) [inline all.json:12]: Ned to look at this one. Inconclusive on the url scan overview.

Comment (frankywild) [reply all.json:12]: You are right, I'm gonna delete it.

Comment (dubstard) [reply all.json]: in polkastarter itself a scam haha

**Review by dubstard — APPROVED** [review]
All good after corrections

**Pull request merged**